cleanslate/templates/registration/password_reset_email.html

```diff
@@ -1,2 +1,12 @@
-Someone asked for password reset for email {{ email }}. Follow the link below:
-{{ protocol}}://{{ domain }}{% url 'password_reset_confirm' uidb64=uid token=token %}
\ No newline at end of file
+{% autoescape off %}
```

Comment:
What does this do?

Reply:
It essentially tells the browser that it's safe to render and isn't a malicious popup.